docs/05_features/10_trx_sponsorship.md

```diff
@@ -0,0 +1,41 @@
+---
+content_title: Transaction Sponsorship
+---
```

Comment:
I would actually call this "Resource Payer" rather than "Transaction Sponsorship".  Resource Payer aligns more with the current functionality.

Reply:
oh, I thought it is the other way around obviously.
resource payer, I find it more generic, and even google it now it brings power up articles and resources articles.
I thought the transaction sponsorship is more self descriptive, less generic, and it speaks for itself that this feature is about somebody else paying for the resources.
can you share a bit more thoughts on why you are saying "resource payer" is better aligned with the feature than "trx sponsorship"?

Reply:
The PRD received from the product team mentions both the terms - "_Transaction Sponsorship_" and "_Resource Payer._"

Probably we can check with the Product team which term they would prefer to use for the docs and general marketing for this feature. 

If they are inclined towards "_Resource Payer_" then we should modify it for clarity and call it 
"_System Resources Payer_" as the system denotes RAM, CPU, and NET. 

https://developers.eos.io/welcome/latest/introduction-to-eosio/core_concepts/#system-resources

Reply:
I just realised that the upgrade protocol feature that has to be enabled for the trx sponsorship/resource payer feature to be enabled is name RESOURCE_PAYER
so I guess resource payer it is